connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceConnector.java

```diff
@@ -538,9 +600,13 @@ Map<String, Config> describeTopicConfigs(Set<String> topics)
             .collect(Collectors.toMap(x -> x.getKey().name(), Entry::getValue));
     }
 
-    Config targetConfig(Config sourceConfig) {
+    Config targetConfig(Config sourceConfig, boolean incremental) {
+// If using incrementalAlterConfigs API, sync the default property with either SET or DELETE action determined by ConfigPropertyFilter::shouldReplicateSourceDefault later.
+// If not using incrementalAlterConfigs API, sync the default property if ConfigPropertyFilter::shouldReplicateSourceDefault returns true.
+// If ConfigPropertyFilter::shouldReplicateConfigProperty returns false, we do not sync the property at all.
```

Comment:
Nit: can we align this with the next line of the method body?
```suggestion
        // If using incrementalAlterConfigs API, sync the default property with either SET or DELETE action determined by ConfigPropertyFilter::shouldReplicateSourceDefault later.
        // If not using incrementalAlterConfigs API, sync the default property if ConfigPropertyFilter::shouldReplicateSourceDefault returns true.
        // If ConfigPropertyFilter::shouldReplicateConfigProperty returns false, we do not sync the property at all.
```